applications/tari_console_wallet/src/cli.rs

```diff
@@ -109,6 +109,7 @@ pub enum CliCommands {
     FinaliseShaAtomicSwap(FinaliseShaAtomicSwapArgs),
     ClaimShaAtomicSwapRefund(ClaimShaAtomicSwapRefundArgs),
     PublishConstitutionDefinition(PublishDefinitionArgs),
+    PublishContractUpdateProposal(PublishUpdateProposalArgs),
```

Comment:
[We're refactoring all the contract related commands to subcommands](https://github.com/tari-project/tari/blob/ed1d7b38cce49538d2a504ccd8381eebf3e1b6d7/applications/tari_console_wallet/src/cli.rs#L209-L216) and this one should probably be included there as well. It'll likely have some conflicts with the refactoring [here as well](https://github.com/tari-project/tari/pull/4185)

Reply:
Thanks for the info, I just adapted the PR to the new command organisation (which is much cleaner IMO)